src/motor_hardware.cc

```diff
@@ -284,6 +344,59 @@ void MotorHardware::sendParams() {
         commands.push_back(winsize);
     }
 
+    if (cycle == 5 && (pid_params.max_pwm != prev_pid_params.max_pwm)) {
```

Comment:
All of this seems to be duplicated from the individual functions that you made for these parameters?

We only ever need to send this data once, so I don't see why it needs to be with all this dynamic reconfigure stuff here

Reply:
that part of the code confused me basically so I did those to be like the other pid parameters.   These indeed do not need to be dynamically changed so if that is the purpose of code in MotorHardware::sendParams  I'm fine with taking it out and will do so.